automatic/flashplayeractivex/update.ps1

```diff
@@ -1,7 +1,9 @@
-
+﻿
 import-module au
+. "$PSScriptRoot\..\..\scripts\Get-Padded-Version.ps1"
 
 $releases = "https://get.adobe.com/en/flashplayer/" # URL to for GetLatest
+$padVersionUnder = '24.0.1'
```

Comment:
@AdmiringWorm the question was really about this.  This is currently set at 24.0.1.  Who or what is responsible for changing this, once the maximum version number allowed/possible goes above this?